src/components/PageSwitch.tsx

```diff
@@ -78,7 +77,8 @@ const PageSwitch = ({
     [locationSearch, navigate]
   )
 
-  const totalNumberOfPages = totalNumberOfElements && Math.ceil(totalNumberOfElements / elementsPerPage)
+  const totalNumberOfPages =
+    totalNumberOfElements && Math.ceil(totalNumberOfElements / (elementsPerPage || defaultElementsPerPage))
```

Comment:
@mvaivre some times, `elementsPerPage` was `0`. Dividing by `0` caused the `Infinity`.